### PR TITLE
feat(view): add 'view' command

### PR DESCRIPTION
**Pull request by larsgw**
As seen in [https://npm.community/t/3088](https://npm.community/t/tink-implement-tink-view/3088?u=larsgw).

### CODE REVIEW

File: lib/commands/view.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const npmlog = require('npmlog')
```
Comment (legodude17) [inline lib/commands/view.js:3]: I'm pretty sure you could use `libnpm.log`.

**Review by zkat — APPROVED** [review]
holy shit this works so nice. Good job!

File: lib/components/view.jsx [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+  }
+
+  items = items.map((prop) =>
+    <span>
+      {prop}: <PackageValue field={`${field}.${prop}`} value={value[prop]}/>
+    </span>
+  )
+
+  if (!props && items.length > maxItems) {
+    items = items.slice(0, --maxItems).concat(
```
Comment (legodude17) [inline lib/components/view.jsx:64]: Why are you decrementing maxItems here?

File: lib/commands/view.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+const { PackageView, PackageFields } = require('../components/view.jsx')
+
+const View = module.exports = {
+  command: 'view [<pkg>[@<version>]] [<field>...]',
+  aliases: ['v', 'info', 'show'],
+  describe: 'Show information about a package',
+  builder (yargs) {
+    return yargs.help().alias('help', 'h').options(View.options)
+  },
+  options: Object.assign(require('../common-opts'), {}),
```
Comment (legodude17) [inline lib/commands/view.js:16]: Wouldn't it make more sense if these arguments were reversed?

**Review by legodude17 — COMMENTED** [review]
I'm so excited for this!

**Comment by legodude17** [comment]
Whoops started that earlier today.

File: lib/components/view.jsx (hunk 2 above)
Comment (larsgw) [reply lib/components/view.jsx:64]: That should probably be done more explicitly, but basically: if the max amount of rows is 6, there should be 5 items and one line explaining there are more. I could also have increased the `maxItems` check by one, but both are wrong, and I found this one more practical, as `maxItems` now always limits the rows.

File: lib/commands/view.js (hunk 3 above)
Comment (larsgw) [reply lib/commands/view.js:16]: Yep, don't know why I did it that way... Not that it matters, at the moment.

**Comment by larsgw** [comment]
@legodude17 No problem, there might be a refactor (to use Ink 2) coming either way, so feedback is appreciated.

File: lib/components/view.jsx (hunk 2 above)
Comment (legodude17) [reply lib/components/view.jsx:64]: Is there a reason you use `--maxItems` and not `maxItems - 1`?

Comment (larsgw) [reply lib/components/view.jsx:64]: Because it's referenced once more (again, not should have done that more explicitly).

**Pull request merged**